test/no-single-promise-in-promise-methods.mjs

```diff
@@ -6,6 +6,8 @@ const {test} = getTester(import.meta);
 // `await`ed
 test.snapshot({
 	valid: [
+		'const results = await Promise.all([promise])',
+		'results = await Promise.all([promise])',
```

Comment:
I don't think this should be valid. The problem mentioned in https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2385 is with the auto fix, not with what the rule reports on.

Reply:
It is corrected.